components/raftstore/src/store/fsm/store.rs

```diff
@@ -2817,10 +2811,42 @@ impl<'a, EK: KvEngine, ER: RaftEngine, T: Transport> StoreFsmDelegate<'a, EK, ER
                 .insert(region.get_id(), peer.peer.read_progress.clone())
                 .is_some()
         {
-            panic!(
-                "key conflicts while insert region {:?} into store meta",
-                region
+            error!(
```

Comment:
panic here

Reply:
let's use `unwrap` for these inserts

Reply:
we expect they return none, kind of opposite of what the `unwrap` does?